PowerFGT/Public/cmdb/interface.ps1

```diff
@@ -0,0 +1,77 @@
+﻿
+
+function Get-FGTInterface {
+    
+  <#
+      .SYNOPSIS
+      Get list of all interfaces
+
+      .DESCRIPTION
+      Get list of all interface
+
+      .EXAMPLE
+      Get-FGTInterface
+
+      Get list of all interface
+
+  #>
+
+
+        $reponse = Invoke-FGTRestMethod -uri 'api/v2/cmdb/system/interface' -method 'GET'
+        $reponse.results
```

Comment:
add parameter (like -name or -match) for search, found a interface (look FGTAddress #31